Day1/index.js

```diff
@@ -0,0 +1,42 @@
+const {sum,subtract,multiply,divide} = require('./lib/math');
+
+
+
+const fs = require("fs");
+
+
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.question("Enter the first number: ", (firstNumber) => {
+  rl.question("Enter the second number: ", (secondNumber) => {
+    const parsedFirst = parseFloat(firstNumber);
+    const parsedSecond = parseFloat(secondNumber);
+    console.log(`You entered ${parsedFirst} and ${parsedSecond}. And result of add , subtract, multiply, divide operrations are following:`);
+    const sumOutput = sum([parsedFirst, parsedSecond])
+    const subtractOutput = subtract(parsedFirst, parsedSecond)
+    const multiplyOutput = multiply(parsedFirst, parsedSecond)
+    const divideOutput = divide(parsedFirst, parsedSecond).toFixed(2)
+   console.log(sumOutput,subtractOutput,multiplyOutput,divideOutput);
+ 
+   const data = `
```

Comment:
@anantmishra-successive 
This is not csv formaty. It shouldn't contain colons. 
Secondly, that's not the right way to create a csv file. This makes it dependent on formatting of you code files, which makes it prone to human error. Also this bulks up the code file with more lines of code.

Reply:
You can use a third party library to do this thing.